src/app/reducers/windows.ts

```diff
@@ -18,6 +16,11 @@ export interface ExportWindowState {
   subscriptionUrl: string;
 }
 
+export interface ExportWindowState extends WindowState {
```

Comment:
I fail to see why you had to do this. Firstly, the naming is going to be confusing, considering you're inside the windows reducer but `WindowState` is not the interface for the "window state" but rather for a slice of the export window state. Secondly, this would defeat the purpose of a specific interface for exported data (which I would argue a duplicate window data is). The version and type data can be easily added to the window data and used, rather than a new ambiguous interface. You can look at https://github.com/imolorhe/altair/blob/staging/src/app/services/window.service.ts#L101-L123 for reference.

Reply:
Basically following the implementation in `importDataFromCurl`, you can use the `ExportWindowState` interface for your implementation as well.

**PS:** The `version` and `type` data are there for backward compatibility, in the event there are new data structures for export window state. They are not limiting factors.

Reply:
Cool, I'll fix now